apps/coscrad-frontend/src/components/AllResources/ResourceInfoPresenter/ResourceInfo.presenter.tsx

```diff
@@ -11,22 +20,26 @@ export const ResourceInfoPresenter = ({
  * TODO [https://www.pivotaltracker.com/story/show/183766033
  * We should expose the Schema as part of our own API docs somehow.
  */
-IAggregateInfo<ResourceType>): JSX.Element => {
-    const resourceIndexLink = (
-        <Link to={`/${routes.resources.ofType(resourceType).index}`}>View {label}s</Link>
-    );
-
-    return (
-        <>
-            <h3>
-                {label} ({resourceType})
-            </h3>
-
-            <div className="resource-meta" data-testid={label}>
-                {resourceIndexLink}
-
-                <div>{description}</div>
-            </div>
-        </>
-    );
-};
+IAggregateInfo<ResourceType>): JSX.Element => (
+    <div>
+        <Card>
```

Comment:
I'm not sure if it's an factor of using a card, or some other thing, but these cards are quite large on the page.  It would be nice to have this a little more condensed so we don't have to scroll as much.  Removing the `CardHeader` and just using an `<h4>` tag will help.

Reply:
I'm not sure I agree with this one. It seems like that is what the `CardHeader` is for. I'll leave it to you and Justin, but isn't the card size something that should be done with css?